deployability/modules/testing/tests/helpers/manager.py

```diff
@@ -25,25 +25,33 @@ def install_manager(inventory_path, node_name, wazuh_version) -> None:
             wazuh_version (str): major.minor.patch
 
         """
-        wazuh_version = '.'.join(wazuh_version.split('.')[:2])
         os_name = HostInformation.get_os_name_from_inventory(inventory_path)
 
+        if live == "False":
+            s3_url = 'packages-dev.wazuh.com'
+        else:
+            s3_url = 'packages.wazuh.com'
```

Comment:
You are declaring `live` as type bool in the function parameters but you are using it as a string, use it as a boolean

Reply:
We need to refactor this module, it will be done in DTT2